src/commands/mod.rs

```diff
@@ -914,22 +914,22 @@ pub struct GetInfoResult {
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct GetAddressResult {
     #[serde(deserialize_with = "deser_addr_assume_checked")]
-    address: bitcoin::Address,
-}
-
-#[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct GetLabelsResult {
-    pub labels: HashMap<String, String>,
+    pub address: bitcoin::Address,
+    pub derivation_index: u32,
```

Comment:
Any reason for not making it a typesafe `ChildNumber`?

Reply:
Not particular, reason it is only exposure to json.